docker/build-tools/Dockerfile

```diff
@@ -49,13 +49,19 @@ ENV PROTOLOCK_VERSION=v0.14.0
 ENV PROTOTOOL_VERSION=v1.8.0
 ENV SHELLCHECK_VERSION=v0.7.0
 ENV UPX_VERSION=3.95
+ENV DOCKER_VERSION=18.06.1
 
 ENV OUTDIR=/out
 WORKDIR /tmp
 
 # Update distro and install dependencies
 # hadolint ignore=DL3008
 RUN apt-get update && apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common \
```

Comment:
what is this needded for?  I comment on line 58 of the various needs for dependencies helps future authors.